_config.yaml

```
@@ -35,7 +35,11 @@ header_pages:
   # as described at https://github.com/AlexsLemonade/training-specific-template#customizing-the-new-repository-for-an-individual-training-workshop
   - workshop/SCHEDULE.md
   - workshop/workshop-structure.md
+  - workshop/software-setup.md
   - workshop/workshop-materials.md
+  - workshop/resources-for-consultation-sessions.md
+  - workshop/posting-errors-guidelines.md
+  - workshop/working-with-your-own-data.md
```

This page is in a slightly different place from the others for whatever reason. I see that you changed the `nav_title` in a separate PR, but we should put that change in the same branch/PR as this one.
```suggestion
  - working-with-your-data/working-with-your-own-data.md
```

---

Okay **@jashapiro**! I attempted to move that change to this branch, but not totally sure if I did that right?

---

Looks good! I think you can close the https://github.com/AlexsLemonade/2021-september-training/pull/19 PR now

---

Done, thanks!